herbridge/main/serializers.py

```diff
@@ -106,14 +106,20 @@ def to_representation(self,obj):
             lon = obj.geom.coords[0]
         else:
             lat,lon = None,None
+            
+        if not obj.captureDate:
+            captureDate = None
+        else:
+            captureDate = obj.captureDate.timestamp()
 
         return {
             'id':obj.pk,
             'url':obj.image.url,
             'thumbnailUrl':obj.thumbnail.url,
             'latitude':lat,
             'longitude':lon,
-            'captureDate':obj.captureDate.timestamp(),
+            'captureDate':captureDate,
+            'caption':obj.caption
```

Comment:
can you do trailing commas on collections in python?

Reply:
yes, and I usually do actually. started doing that after reading https://github.com/ambv/black#trailing-commas. One could be added here after `obj.caption`.